frontend/src/lib/components/metadata/cells/TagCell.svelte

```diff
@@ -201,7 +201,18 @@
 								</button>
 							{/if}
 							<button
-								class="py flex w-20 items-center px-2 hover:bg-grey-lighter"
+								class="py flex w-24 items-center px-2 hover:bg-grey-lighter"
+								on:click={(e) => {
+									e.stopPropagation();
+									showOptions = false;
+									navigator.clipboard.writeText(tag.dataIds.toString());
```

Comment:
I think this should copy with the two brackets `[ ]` around it so you can just directly paste into Python.